config/package-bundle/contents/values.yml

```diff
@@ -0,0 +1,20 @@
+#@data/values-schema
```

Comment:
should name values-schema files as values-schema.yml @pivotaljohn wdyt?

Reply:
sure... or `schema.yml`

But not `values.yml`... that is too often either a Data Values overlay or a Data Values File (i.e. input to `--data-values-file`).